lib/api.js

```diff
@@ -1334,6 +1334,8 @@ function sendMetric (name, options) {
  *                  value is associated with this type)
  * 2) value-based - a specific value (or sum of values if count > 1).
  *
+ * The metrics submitted are aggregated by metric name and tag(s), then
+ * sent on interval every 60 seconds.
```

Comment:
`on interval` needed? I think the sentence would be fine just saying "The metrics submitted are aggregated by metric name and tag(s), then sent every 60 seconds."

Reply:
That's true!  Updated.